setup.py

```diff
@@ -269,6 +270,7 @@ def setup_requirements(requirements_path, module_names, strict_bounds,
         )
     return module_lines
 
+
```

Comment:
I'm guessing this was added because of a `flake8` error?

Reply:
Yup, exactly